code/modules/emotes/emote_sounds.dm

```diff
@@ -0,0 +1,52 @@
+/*
+* Declare for storing species and gender specific emote sounds and make them overridable between species. 
+* Also the soundlist is loaded from a json array that sorts lists of sounds by emote name and optionally gender.
+*/
+
+//Mainly used for debugging and verifying everything loaded properly
+var/global/list/decl/emote_sounds/all_emote_decl = list()
+
+//merging proc for the merge assoc list proc
+/proc/merge_replace_if_not_null(var/a, var/b)
+	return b? b : a
+
+//A set of sounds for a set of emotes
+/decl/emote_sounds
+	var/list/soundbank = list() // 2d array, first is emote_name, second level contains sounds list by gender, or just a soundlist if there's no need for gender
+
+/decl/emote_sounds/New()
+	..()
+	init_soundlist()
+	all_emote_decl += src
+
+/decl/emote_sounds/Destroy()
+	all_emote_decl -= src
+	. = ..()
+
+/decl/emote_sounds/proc/init_soundlist()
+	load_sound_config(sound_config_paths())
```

Comment:
This could be done in Initialize(), seems like it's only called in New() currently.

Reply:
Yeah, the main reason why I'm splitting it into another proc is for overriding. Its nearly impossible to completely override stuff that ends up being in initialize without having to copy paste the code you need to still run. While putting it in a separate proc you can easily completely override it without breaking init or copy pasting crap. 
I deal with that everyday currently when overriding stuff for saving soo, I thought I'd do something about it XD